docs/guides/06-advanced-guide-cartojs.md

```diff
@@ -0,0 +1,268 @@
+
+### Adding a Category Widget
+Adding a Category Widget is as simple as including `<as-category-widget>` tag within our `<body>` tag, setting the options you prefer.
+
+```html
+<body>
+  <as-category-widget
+    heading="Populated places"
+    description="Sum of the population grouped by country">
+  </as-category-widget>
+</body>
+```
+
+Although the widget is present in our layout, it lacks some data to display. Let's populate it with data from CARTO.js dataviews.
+
+### Populating Category Widget with CARTO.js dataviews
+Dataviews are the way to get the data which is showed in the map. That way we'll be able to show it in our Category widget.
+Listening to `dataChanged` event will allow us to set the proper categories into category widget.
+
+```js
+const categoryWidget = document.querySelector('as-category-widget');
+const dataView = new carto.dataview.Category(source, 'adm0name', {
+  operation: carto.operation.SUM,
+  operationColumn: 'pop_max'
+});
+
+dataView.on('dataChanged', function (newData) {
+  categoryWidget.categories = newData.categories;
+});
+
+client.addDataview(dataView);
+```
+
+### Filtering map geometries with Category widget
+You can be aware of selected categories in your category widget by listening `categoriesSelected` event. It will fire an event whenever the selected categories change.
+
+Inside the event handler, there will be a `event` parameter containing the selected categories within `event.detail` property. So that we can tell CARTO.js which categories are the ones to be shown in the map.
+
+Leveraging CARTO.js' Source Filters, we can create a filter, attach it to the source and filter the points out in our map.
+
+```js
+const selectedCountries = new carto.filter.Category('adm0name', {});
+source.addFilter(selectedCountries);
+
+categoryWidget.addEventListener('categoriesSelected', (event) => {
+  selectedCountries.setFilters({ in: event.detail });
+});
+```
+
+## Example HTML
+```html
+<!DOCTYPE html>
+<html>
+  <head>
+      <meta charset="UTF-8">
+      <meta name="viewport" content="width=device-width,initial-scale=1.0">
+      <meta http-equiv="X-UA-Compatible" content="ie=edge">
+
+      <!-- Include CSS  -->
+      <link rel="stylesheet" href="https://libs.cartocdn.com/airship-styles/v1.0.0-alpha.37/airship.css">
+      <!-- Include Icons -->
+      <link rel="stylesheet" href="https://libs.cartocdn.com/airship-icons/v1.0.0-alpha.37/icons.css">
+      <!-- Include Web Components -->
+      <script src="https://libs.cartocdn.com/airship-components/v1.0.0-alpha.37/airship.js"></script>
+
+      <!-- Include Leaflet -->
+      <script src="https://unpkg.com/leaflet@1.3.1/dist/leaflet.js"></script>
+      <link href="https://unpkg.com/leaflet@1.3.1/dist/leaflet.css" rel="stylesheet">
+
+      <script src="https://libs.cartocdn.com/carto.js/v4/carto.min.js"></script>
+
+      <style>
+        body {
+          margin: 0;
+          padding: 0;
+        }
+
+        .widgets {
+          padding: 10px;
```

Comment:
I think we should have a default padding

Reply:
Why can't we use a utility class instead of adding a new class here?